src/XactionInitiator.h

```diff
@@ -22,7 +22,6 @@ class XactionInitiator {
         initHtcp = 1<< 5, ///< HTCP client
         initIcp = 1 << 6, ///< the ICP/neighbors subsystem
         initIcmp = 1 << 7, ///< the ICMP RTT database (NetDB) neighbors exchange subsystem
-        initAsn = 1 << 8, ///< the ASN db subsystem
```

Comment:
It is not visible in the default diff, but this is not the first unmarked hole in Initiator values. Thus, I think it is OK to just delete this value -- no marking or shifting is required. We may reuse this and another spot when adding new values.

This comment does not request any PR changes.